docs/upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -79,4 +85,20 @@ There you can find links to upgrade notes for other versions too.
         +       arguments:
         +           $useInlineEditation: false
         ```
+- add support to display date-time values in different timezone ([#1343](https://github.com/shopsys/shopsys/pull/1343))
+    - you can read more about how to [work with display timezone in documentation](/docs/introduction/working-with-date-time-values.md)
+    - you can delete test `tests/ShopBundle/Functional/Twig/DateTimeFormatterExtensionTest.php` as it was moved to the FrameworkBundle, if you don't test any your specific use-case
```

Comment:
~don't~ do not

Reply:
fixed in ad2dcfae1808de200ae563f458e1aa9fc60420a2